apps/cli/src/node/docker-compose.yml

```diff
@@ -82,7 +82,7 @@ services:
                 condition: service_completed_successfully
         volumes:
             - blockchain-data:/opt/cartesi/share/deployments:ro
-            - ./.sunodo/image:/var/opt/cartesi/machine-snapshots/0_0
+            - ./.sunodo/image:/tmp/machine-snapshots/0_0:ro
```

Comment:
do we need `0_0` here?

Reply:
I wanted to be explicit. 

But it's not necessary and we change the up.sh script.